src/client/src/orangehrmPerformancePlugin/pages/KpiSave.vue

```diff
@@ -0,0 +1,186 @@
+        <oxd-form-row>
+          <oxd-grid :cols="2" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                v-model="kpi.title"
+                label="Key Performance Indicator"
+                required
+                :rules="rules.title"
+                autcomplete="off"
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <jobtitle-dropdown
+                v-model="kpi.jobTitle"
+                required
+                :rules="rules.jobTitle"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="4" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                v-model.number="kpi.minRating"
+                label="Minimum Rating"
+                required
+                autcomplete="off"
+                :rules="rules.minRating"
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <oxd-input-field
+                v-model.number="kpi.maxRating"
+                label="Maximum Rating"
+                required
+                autcomplete="off"
+                :rules="rules.maxRating"
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <div class="orangehrm-module-field-row">
+                <oxd-text tag="p" class="orangehrm-module-field-label">
+                  Make Default Scale
+                </oxd-text>
+                <oxd-switch-input v-model="kpi.isDefault" />
+              </div>
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+
+        <oxd-divider />
+        <br />
+        <oxd-form-actions>
+          <required-text />
+          <oxd-button
+            type="button"
+            display-type="ghost"
+            label="Cancel"
+            @click="onCancel"
+          />
+          <submit-button />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import SwitchInput from '@ohrm/oxd/core/components/Input/SwitchInput';
+import {navigate} from '@ohrm/core/util/helper/navigation';
+import {
+  required,
+  shouldNotExceedCharLength,
+  minValueShouldBeLowerThanMaxValue,
+  maxValueShouldBeGreaterThanMinValue,
+  numberShouldBeBetweenMinandMaxValue,
+} from '@ohrm/core/util/validation/rules';
+import {APIService} from '@/core/util/services/api.service';
+import JobtitleDropdown from '@/orangehrmPimPlugin/components/JobtitleDropdown.vue';
+
+const initialKpi = {
+  title: '',
+  jobTitle: null,
+  minRating: null,
+  maxRating: null,
+  isDefault: false,
+};
+
+export default {
+  name: 'KpiSave',
+  components: {
+    'oxd-switch-input': SwitchInput,
+    'jobtitle-dropdown': JobtitleDropdown,
+  },
+  setup() {
+    const http = new APIService(
+      window.appGlobal.baseUrl,
+      'api/v2/performance/kpi',
```

Comment:
can use plural nouns for collection endpoint. 
https://swagger.io/resources/articles/best-practices-in-api-design/